### ISSUE TITLE
Can't download model from model zoo.

### ISSUE THREAD

**Issue by imyhxy** [open-mmlab/mmtracking#74]
Can't download models from [page](https://github.com/open-mmlab/mmtracking/blob/master/configs/mot/deepsort/README.md) 
Error messages:

```xml
<Error>
  <Code>AccessDenied</Code>
  <Message>You have no right to access this object because of bucket acl.</Message>
  <RequestId>5FF404C49C2407EF3DDB78D7</RequestId>
  <HostId>download.openmmlab.com</HostId>
</Error>
```

**Comment by OceanPang** [comment]
Thanks for the report. We will fix the link soon. 
You can refer to the URLs in the configs that are work now.

**Comment by OceanPang** [comment]
Fixed in #76
